addons/sale/views/sale_portal_templates.xml

```diff
@@ -26,6 +32,20 @@
         <div id="portal_alert_category" position="inside">
             <t t-call="portal.portal_docs_entry">
                 <t t-set="bg_color" t-value="'alert alert-primary align-items-center'"/>
+                <t t-set="placeholder_count" t-value="'overdue_invoice_count'"/>
+                <t t-set="title">Invoices to pay</t>
+                <t t-set="url" t-value="'/my/invoices?filterby=overdue_invoices'"/>
+                <t t-set="show_count" t-value="True"/>
+                <t t-set="html_children">
+                    <button onclick="event.preventDefault(); window.location.href='/my/invoices/overdue'" class="btn btn-secondary ms-auto">
+                        Pay Now
+                    </button>
+                </t>
+            </t>
+        </div>
+        <div id="portal_alert_category" position="inside">
+            <t t-call="portal.portal_docs_entry">
+                <t t-set="bg_color" t-value="'alert alert-primary'"/>
```

Comment:
How is the diff of this file related to sale ? :thinking: 
Seems like it's something that should be in `account_payment` or `account` instead.